docs/real_world_fl.rst

```diff
@@ -14,6 +14,8 @@ help gather information to provision a project and distribute startup kits, see
 For more details on what you can do with apps with custom components and
 the flexibility that the Controller and Worker APIs bring, see the :ref:`programming_guide`.
 
+For setting up authorization policies, see :ref:`federated authorization <federated_authorization>`.
```

Comment:
Since there was a link to this from an external blog, I re-added this file back with a link to where this has been moved under the security section under identity_security.rst